tests/test_conda_local_channel.py

```diff
@@ -0,0 +1,16 @@
+"""
+Tests for making sure the conda_local_channel fixture functions as we expect
+"""
+
+import urllib.request
+import json
+
+
+def test_conda_channel(conda_local_channel):
+    """Verify the conda channel server is working."""
+    url = f"{conda_local_channel}/noarch/repodata.json"
+    with urllib.request.urlopen(url) as response:
+        repodata = json.loads(response.read())
```

Comment:
response.json() should exist https://urllib3.readthedocs.io/en/stable/reference/urllib3.response.html#urllib3.response.BaseHTTPResponse.json